apps/src/applab/applab.js

```diff
@@ -1315,7 +1329,38 @@ Applab.onPropertyChange = function(element, name, value) {
       // element.checked represents the current state, the attribute represents
       // the serialized state
       element.checked = value;
-      element.setAttribute('checked', value ? 'checked' : null);
+      if (value) {
+        element.setAttribute('checked', 'checked');
```

Comment:
for radio buttons, this should also uncheck the other radio buttons in the group